src/index.ts

```diff
@@ -11,18 +11,20 @@ import { version } from "../package.json";
 
 const { argv } = process;
 
-type Config = {
+export type Config = {
   help: boolean,
   include: string[],
   version: boolean,
   write: boolean,
+  renameOnlyChild: boolean;
```

Comment:
https://github.com/benawad/destiny/pull/89#discussion_r386085311
```suggestion
  avoidSingleFile: boolean;
```